src/09-word-problems.js

```diff
@@ -127,7 +175,32 @@ function compareLocations(address1, address2) {}
     //>   },
     //> ];
  */
-function gradeAssignments(assignments) {}
+function gradeAssignments(assignments) {
```

Comment:
Let's really step through this code one line at a time.  We have a lot going on here and we are close.